src/main/java/seedu/duke/commands/EditCommand.java

```diff
@@ -0,0 +1,120 @@
+package seedu.duke.commands;
+
+import seedu.duke.exceptions.ModHappyException;
+import seedu.duke.exceptions.NoSuchModuleException;
+import seedu.duke.tasks.Module;
+import seedu.duke.tasks.ModuleList;
+import seedu.duke.tasks.Task;
+import seedu.duke.tasks.TaskList;
+import seedu.duke.util.StringConstants;
+
+import java.util.Objects;
```

Comment:
I think we have to import java library before custom classes

Reply:
yup, changed the order